recipes/libpqxx/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+# Project overrides
```

Comment:
Could using a newer SDK be enough? See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=macosx_sdk_version#requiring-newer-macos-sdks

Reply:
It doesn't seem like it: 
```
FAILED: test/CMakeFiles/runner.dir/test26.cxx.o 
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang++ -DPQXX_SHARED -I$PREFIX/include/server -Iinclude -I../include -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -std=c++14 -fmessage-length=0 -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/libpqxx-7.4.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -O3 -DNDEBUG -isysroot /Applications/Xcode_12.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk -mmacosx-version-min=10.9 -std=c++17 -MD -MT test/CMakeFiles/runner.dir/test26.cxx.o -MF test/CMakeFiles/runner.dir/test26.cxx.o.d -o test/CMakeFiles/runner.dir/test26.cxx.o -c ../test/test26.cxx
../test/test26.cxx:47:21: error: 'value' is unavailable: introduced in macOS 10.13
      conversions[y.value()] = To4Digits(y.value());
```

Shall I go back to what I had before?

Reply:
Yes but please include a comment why .

Reply:
Cool, just done this @xhochy.